projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -247,6 +248,32 @@ public static function style( $style ) {
 	public static function style_on() {
 		return self::style( true );
 	}
+	/**
+	 * Adds a quick link to the admin bar for the contact form entries.
+	 *
+	 * @param \WP_Admin_Bar $admin_bar The admin bar object.
+	 */
+	public static function add_quick_link_to_admin_bar( \WP_Admin_Bar $admin_bar ) {
+
+		if ( ! current_user_can( 'edit_pages' ) ) {
+			return;
+		}
+
+		$url = ( new Dashboard_View_Switch() )->get_forms_admin_url();
+
+		$admin_bar->add_menu(
+			array(
+				'id'     => 'jetpack-forms',
+				'parent' => null,
+				'group'  => null,
+				'title'  => __( 'Form Responses', 'jetpack-forms' ),
+				'href'   => $url,
+				'meta'   => array(
+					'title' => __( 'Form Responses', 'jetpack-forms' ),
+				),
```

Comment:
Is there really a need for this? 

Some a11y feedback I've seen in the past is that links, buttons, etc. shouldn't just repeat their textual content. My understanding is that users relying on screen readers often then just get duplicate text being read out. Visually, a tooltip saying the same thing as the textual label doesn't add a lot of value either.

I'd suggest we either:
1. Remove this `meta.title` property
2. Make the title text here more meaningful and add additional context for all users

Reply:
Good call.